app/models/pipeline.py

```diff
@@ -39,7 +48,8 @@ def get_id(self):
     #simple lookup using query id, returns none if not in db
     def get_by_username(username):
```

Comment:
maybe would want to lookup using user_id instead since it's unique? or we can make username unique

Reply:
gotcha. idk how to check whether a username appears in the db already or not but im sure we can figure it out pretty easily.